dask_sql/physical/rex/core/call.py

```diff
@@ -629,6 +629,8 @@ def extract(self, what, df: SeriesOrScalar):
             return df.week
         elif what in {"YEAR", "YEARS"}:
             return df.year
+        elif what == "DATE":
+            return df.date()
```

Comment:
Random question - is `ExtractOperation` still accessible through queries? Running through the test suite, it seems like the only time that this function is entered is through `test_dates` when it's called explicitly, but it seems like in all other cases we would use `DatePartOperation`

Reply:
It doesn't look like it - none of the combinations of `EXTRACT(DATE FROM d)`, `EXTRACT(YEAR FROM d)`, etc. queries I tested out used it.

When I remove the `ExtractOperation` class entirely, it doesn't appear to cause any of the tests in `test_rex` to fail.

Should we still keep the class? I could also open another PR for removing it.

Reply:
Yeah removing it in a follow up makes sense to me, thanks @sarahyurick!

Reply:
Sounds good, opened https://github.com/dask-contrib/dask-sql/pull/1011